src/Lykke.Logs/Loggers/LykkeSanitizing/SanitizingLog.cs

```diff
@@ -26,7 +26,12 @@ public ISanitizingLog AddSanitizingFilter(Regex pattern, string replacement)
             return this;
         }
 
-        public string Sanitize(string value) => _options.Filters.Aggregate(value, (a, p) => p.Pattern.Replace(a, p.Replacement));
+        public string Sanitize(string value)
+        {
+            return string.IsNullOrWhiteSpace(value)
```

Comment:
If value is null, better to return null, otherwise it could break something

Reply:
Done, now return value.